### ISSUE TITLE
Translations doesn't change 

### ISSUE THREAD

**Issue by MnlLBD1** [joedixon/laravel-translation#204]
Hello, 

First, thank you for this great package and the hard work; 

Ths issue I am having is that translations don't save, When a text is translated I have an X instead of the green button. 

Do you have any idea what it could be about?

**Comment by legreco** [comment]
same problem here. Did you solve it?

**Comment by MnlLBD1** [comment]
Yes by giving permissions on ( database and resources/lang ) folders

**Comment by AnniDineva** [comment]
> Yes by giving permissions on ( database and resources/lang ) folders

Hi, how or where do you tho that?  I ask because I have the seem problem and I don't know where to give this permisions
